### PR TITLE
Startup commands and README update

### PR DESCRIPTION
**Pull request by kodyang**
## Description
Our app is hard to start. I made a long ugly command that does the dirty work in one command. There is still some manual work replacing code.

### CODE REVIEW

File: private-templates-service/README.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+  authenticationProvider: new AzureADProvider(),
+  storageProvider: new InMemoryDBProvider(),
+}
+const client: TemplateServiceClient = TemplateServiceClient.init(mongoClient);
+app.use("/template", client.expressMiddleware());
+app.use("/user", client.userExpressMiddleware());
+```
+
+**5. In the `adaptivecards-templates/private-templates-service/server/app.ts` file, replace ` "#{DB_CONNECTION_TOKEN}#"` with the real DB connection string. Make sure to NOT commit this file. Ask a dev for this string!**
+
```
Comment (gracez72) [inline private-templates-service/README.md]: Add a step saying to comment out line 50 in `AzureADProvider.ts` 
```
result = result && "#{CLIENT_ID_TOKEN}#" === decodedToken.payload.aud;
```

File: private-templates-service/server/package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "dev": "concurrently \"npm run server\" \"npm run client\"",
     "server": "nodemon --watch 'src/**/*.ts' --ignore 'src/**/*.spec.ts' --exec ts-node src/server.ts | bunyan",
     "client": "cd ../client && npm start",
-    "link-client": "cd ../adaptivecards-templating-client/typescript && npm install && npm run build && npm link && cd ../../client && npm link adaptive-templating-service-typescript-node && npm install && cd src && npm link adaptive-templating-service-typescript-node"
+    "link-client": "cd ../adaptivecards-templating-client/typescript && npm install && npm run build && npm link && cd ../../client && npm link adaptive-templating-service-typescript-node && npm install && cd src && npm link adaptive-templating-service-typescript-node",
```
[the remark this answers is not in the document]
Comment (kodyang) [reply private-templates-service/server/package.json]: Which one do you link ? In theory just `client`, but we've been doing both this whole time lol

File: private-templates-service/README.md (hunk 1 above)
Comment (kodyang) [reply private-templates-service/README.md]: Wait, do we want to replace the string or just comment it out ?

Comment (gracez72) [reply private-templates-service/README.md]: We can add replace it here, I'll add a section below for troubleshooting this line in a later PR!

File: private-templates-service/server/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (gracez72) [reply private-templates-service/server/package.json]: I've been linking under client/ only

File: private-templates-service/README.md [hunk 3]
```diff
@@ -1,3 +1,49 @@
...
+const mongoClient: ClientOptions = {
+  authenticationProvider: new AzureADProvider(),
+  storageProvider: new InMemoryDBProvider(),
+}
+const client: TemplateServiceClient = TemplateServiceClient.init(mongoClient);
+app.use("/template", client.expressMiddleware());
+app.use("/user", client.userExpressMiddleware());
+```
+
+**5. In the `adaptivecards-templates/private-templates-service/server/app.ts` file, replace ` "#{DB_CONNECTION_TOKEN}#"` with the real DB connection string. Make sure to NOT commit this file. Ask a dev for this string!**
```
Comment (daniellemott) [inline private-templates-service/README.md]: If we're suggesting that they use the inMemoryDB provider, I think we can remove step #5.

File: private-templates-service/README.md [hunk 4]
```diff
@@ -1,3 +1,49 @@
...
+-   Git
+-   [Node v12](https://nodejs.org/en/download/)
+
+**1. Clone the reponsitory.**
+
+**2. Switch to the desired branch. The latest build is on `dev`**
+
+**3. `cd adaptivecards-templates/private-templates-service/server` and run `npm run init-app`. This installs and links dependencies.**
+
+**4. Open `server/app.ts` and replace mongoose code lines 39 - 64 with the following:**
```
Comment (daniellemott) [inline private-templates-service/README.md]: Do you mean lines 29 - 64?

**Pull request merged**